conda/multisheller/matlab-whole-body-simulator_activate.msh

```diff
@@ -0,0 +1,21 @@
+if_(is_set("COMSPEC")).then_([
+  sys.list_append("YARP_DATA_DIRS", path.join(env("CONDA_PREFIX"), "Library\\share\\RRbot")),
+  sys.list_append("ROS_PACKAGE_PATH", path.join(env("CONDA_PREFIX"), "Library\\share")),
+  sys.list_append("AMENT_PREFIX_PATH", path.join(env("CONDA_PREFIX"), "Library\\share")),
+	sys.list_append("GAZEBO_MODEL_PATH", path.join(env("CONDA_PREFIX"), "Library\\share\\RRbot\\robots")),
```

Comment:
To avoid problems it is better to avoid to mix spaces and tabs. The tabs in other msh files are probably my fault, but at least in newly introduced msh files I think we can be consistent and use spaces.

Reply:
The tabs were probably a result of a copy paste, sorry for that. Will chase them down in my changes.

Reply:
Done in https://github.com/robotology/robotology-superbuild/pull/689/commits/b24f8ee65e8a7c741019a015316c46474cc356b2.